README.md

```diff
@@ -26,8 +26,13 @@ Please checkout [ML on GCP](https://github.com/GoogleCloudPlatform/ml-on-gcp) gu
 * [Reddit](reddit_tft) - Predict the score of a Reddit thread using a wide and deep model
 * [Cifar10](https://github.com/tensorflow/models/tree/master/tutorials/image/cifar10_estimator) - Classify image content (training on multiple GPUs)
 
-### TPU (Contrib)
-* [Resnet TPU](tpu)
+### TPU
```

Comment:
Let's make this a heading level 2, since it's applicable to Cloud TPU on GCE rather than just ML Engine:

```
## TPU
```

Reply:
Good point. Changed it to Cloud TPU instead.